automathor.py

```
@@ -0,0 +1,202 @@
+    todo: str
+    text: str
+    lines: List[AnnotatedLine]
+
+    def __repr__(self):
+        """Return a string representation of the context."""
+        return f"{self.filename}:{self.line}\n{self.author_email} {self.datetime}\n{self.commit}\n"
+
+    def render(self):
+        """Render the context as a markdown file."""
+        template_loader = jinja2.FileSystemLoader(searchpath="./")
+        template_env = jinja2.Environment(loader=template_loader)  # nosec B701
```

Ruff (0.8.2)

By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)